src/components/Search/SearchAutocompleteList.tsx

```diff
@@ -57,6 +56,9 @@ type SearchAutocompleteListProps = {
     /** Value of TextInput */
     autocompleteQueryValue: string;
 
+    /** Callback to trigger search action * */
+    handleSearch?: (value: string) => void;
```

Comment:
Is there cases where we don't need to pass the search handle? If not let's make this required